src/api/test-mocks/index.ts

```diff
@@ -36,6 +36,7 @@ export const mockExporterCompany = {
 
 export const mockBuyer = {
   companyOrOrganisationName: 'Mock buyer',
+  // buyerLocation: mockCountries[0].name,
```

Comment:
remove?

Reply:
Great spot! Since everything is green can I do this in the next PR? Removed it locally already